Exercise_4.py

```diff
@@ -7,15 +7,50 @@ def __init__(self, data):
         self.right = None
           
 """ Inorder traversal of a binary tree"""
-def inorder(temp): 
-  
-    
+def inorder(temp):
+    # Recursive Solution
+    # if temp: 
+    #     inorder(temp.left)
+    #     print (temp.key, end = " ")
+    #     inorder(temp.right)
+
+    # Iterative solution
+    stack = []
+    curr = temp
+    while True:
```

Comment:
Good work on implementing the iterative solution

Reply:
Thanks!